### PR TITLE
add links to dashboards and dankrad's article to client diversity page

### PR DESCRIPTION
**Pull request by jmcook1186**
## Description

Since this page was published several new client diversity dashboards have become available, and Dankrad from the EF posted a detailed description of the risks of going through the merge with the majority client.

I have added links to these resources in this PR.

### CODE REVIEW

**Review by samajammin — APPROVED** [review]

File: src/content/developers/docs/nodes-and-clients/client-diversity/index.md [hunk 1]
```diff
@@ -77,9 +77,24 @@ Addressing client diversity requires more than individual users to choose minori
...
+## Client diversity dashboards
+
+There are now several dashboards giving real time client diversity statistics for the execution and consensus layers, including:
+
+Consensus Layer:
+
+[Rated.network](https://www.rated.network/)
+[clientdiversity.org](https://clientdiversity.org/)
+[pools.invis.cloud](https://pools.invis.cloud/)
+[slashed.info](https://www.slashed.info/)
```
Comment (minimalsm) [inline src/content/developers/docs/nodes-and-clients/client-diversity/index.md:89]:
```suggestion
- [Rated.network](https://www.rated.network/)
- [clientdiversity.org](https://clientdiversity.org/)
- [pools.invis.cloud](https://pools.invis.cloud/)
- [slashed.info](https://www.slashed.info/)
```

File: src/content/developers/docs/nodes-and-clients/client-diversity/index.md [hunk 2]
```diff
@@ -77,9 +77,24 @@ Addressing client diversity requires more than individual users to choose minori
...
+
+Execution Layer:
+[Ethernodes](https://ethernodes.org/)
```
Comment (minimalsm) [inline src/content/developers/docs/nodes-and-clients/client-diversity/index.md:92]:
```suggestion

- [Ethernodes](https://ethernodes.org/)
```

File: src/content/developers/docs/nodes-and-clients/client-diversity/index.md [hunk 3]
```diff
@@ -77,9 +77,24 @@ Addressing client diversity requires more than individual users to choose minori
...
+There are now several dashboards giving real time client diversity statistics for the execution and consensus layers, including:
+
+Consensus Layer:
+
+[Rated.network](https://www.rated.network/)
+[clientdiversity.org](https://clientdiversity.org/)
+[pools.invis.cloud](https://pools.invis.cloud/)
+[slashed.info](https://www.slashed.info/)
+
+Execution Layer:
```
Comment (minimalsm) [inline src/content/developers/docs/nodes-and-clients/client-diversity/index.md:91]:
```suggestion
**Execution layer:**
```

File: src/content/developers/docs/nodes-and-clients/client-diversity/index.md [hunk 4]
```diff
@@ -77,9 +77,24 @@ Addressing client diversity requires more than individual users to choose minori
 
 Technical users can help accelerate this process by writing more tutorials and documentation for minority clients and encouraging their node-operating peers to migrate away from the dominant clients. Guides for switching to a minority consensus client are available on [clientdiversity.org](https://clientdiversity.org/).
 
+## Client diversity dashboards
+
+There are now several dashboards giving real time client diversity statistics for the execution and consensus layers, including:
```
Comment (minimalsm) [inline src/content/developers/docs/nodes-and-clients/client-diversity/index.md:82]:
```suggestion
Several dashboards give real-time client diversity statistics for the execution and consensus layer.
```

File: src/content/developers/docs/nodes-and-clients/client-diversity/index.md [hunk 5]
```diff
@@ -77,9 +77,24 @@ Addressing client diversity requires more than individual users to choose minori
 
 Technical users can help accelerate this process by writing more tutorials and documentation for minority clients and encouraging their node-operating peers to migrate away from the dominant clients. Guides for switching to a minority consensus client are available on [clientdiversity.org](https://clientdiversity.org/).
 
+## Client diversity dashboards
```
Comment (minimalsm) [inline src/content/developers/docs/nodes-and-clients/client-diversity/index.md:80]:
```suggestion
## Client diversity dashboards {#client-diversity-dashboards}
```

File: src/content/developers/docs/nodes-and-clients/client-diversity/index.md [hunk 6]
```diff
@@ -77,9 +77,24 @@ Addressing client diversity requires more than individual users to choose minori
...
+[pools.invis.cloud](https://pools.invis.cloud/)
+[slashed.info](https://www.slashed.info/)
+
+Execution Layer:
+[Ethernodes](https://ethernodes.org/)
+
 ## Further reading {#further-reading}
 
 - [Client diversity on Ethereum's consensus layer](https://mirror.xyz/jmcook.eth/S7ONEka_0RgtKTZ3-dakPmAHQNPvuj15nh0YGKPFriA)
+- [Ethereum merge: run the majority client at your own peril! - Dankrad Fiest](https://dankradfeist.de/ethereum/2022/03/24/run-the-majority-client-at-your-own-peril.html)
```
Comment (minimalsm) [inline src/content/developers/docs/nodes-and-clients/client-diversity/index.md:97]:
```suggestion
- [Ethereum Merge: Run the majority client at your own peril!](https://dankradfeist.de/ethereum/2022/03/24/run-the-majority-client-at-your-own-peril.html) – _Dankrad Fiest, March 24 2022_
```

**Review by minimalsm — APPROVED** [review]
LGTM :-) Added a few small formatting changes. Merging 🚀

**Pull request merged**